packages/core/src/transports/initialize.ts

```diff
@@ -111,18 +111,9 @@ export function initializeTransports(internalLogger: InternalLogger, config: Con
     transportsToRemove.forEach((transportToRemove) => {
       internalLogger.debug(`Removing "${transportToRemove.name}" transport`);
 
-      const existingTransportIndex = transports.reduce<number | null>(
-        (acc, existingTransport, existingTransportIndex) => {
-          if (acc === null && existingTransport.name === transportToRemove.name) {
-            return existingTransportIndex;
-          }
+      const existingTransportIndex = transports.indexOf(transportToRemove);
```

Comment:
Wouldn't it be better with `.find` instead of `indexOf` given that we don't use the index anywhere?

Reply:
It's used on 122: `transports.splice(existingTransportIndex, 1);`